### ISSUE TITLE
[4.0] Notices in Options should have same Behaviour

### ISSUE THREAD

**Issue by ghost** [joomla/joomla-cms#21401]
### Steps to reproduce the issue
Text "Warning! Path Folder ..." should have same Width and Layout as Example below "Enter the language Tag ..."

![screen shot 2018-08-04 at 10 51 59](https://user-images.githubusercontent.com/8235763/43674460-92e2d0d6-97d4-11e8-8359-b7d7c6490390.jpg)

![screen shot 2018-08-04 at 10 52 40](https://user-images.githubusercontent.com/8235763/43674462-95d77d50-97d4-11e8-92c1-46635f4b588e.jpg)

### System information
- Nightly Build 4.0.0-alpha5-dev
- Multilingual Sample Data (French, German DE, Persian)
- Template: Cassiopeia
- macOS Sierra, 10.13.6
- Firefox 61 (64-bit)
#### CloudAccess.net
- PHP 7.1.15
- MySQLi 5.7.18-cll-lve

**Comment by TobsBobs** [comment]
Issue confirmed!

**Comment by infograf768** [comment]
In the first case it is a spacer, in the second case is it a tip/description.

I.e
```
		<field
			name="spacer1" 
			type="spacer" 
			label="COM_MEDIA_FOLDERS_PATH_LABEL"
			class="text"
		/>
```
vs
```
		<field 
			name="lang_code" 
			type="text"
			label="COM_LANGUAGES_FIELD_LANG_TAG_LABEL"
			description="COM_LANGUAGES_FIELD_LANG_TAG_DESC"
			maxlength="7"
			required="true"
			size="10"
		/>
```

Unless some css changes are done.
For example
```
#component .field-spacer > .control-label {
  width: auto;
}
```
would give

<img width="645" alt="screen shot 2018-08-04 at 17 48 48" src="https://user-images.githubusercontent.com/869724/43678183-b0ebd548-980e-11e8-803e-715edc5a25a6.png">

Which is more readable than the muted small font size tip.

But, we know that back-end template is WIP. So I am not going to make a PR for that.

**Comment by ghost** [comment]
closed as having Pull Request #21425

**Comment by ghost** [comment]
Reopened as PR is closed.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/21401">issues.joomla.org/tracker/joomla-cms/21401</a>.</sub>

**Comment by ghost** [comment]
closed as having Pull Request #22213
